### ISSUE TITLE
Test results of discretization methods

### ISSUE THREAD

**Issue by serenity4** [JuliaGeometry/Meshes.jl#70]
Keeping track of the fact that additional tests should be added for FIST, notably to test the correctness of the result. Additional tests introduced in #66 but eventually not merged in #68 included tests for the output regarding the number of triangles, connectivity coverage and vertex data.

**Comment by serenity4** [comment]
Triangle count and vertex data tests are now implemented via #75. We may want to consider the following tests:
- [ ] Connectivity coverage: check that all points are connected to a `Triangle` in the mesh.
- [ ] Triangulation correctness: check that all triangles are simple.
- [ ] Triangulation quality: We may want to test that several options (not implemented yet) in FIST give actual improvements (particularly if there is a cost to pay for a supposedly better quality).

Regarding this last item, we fist need to have metrics that measure the quality of a triangulation. I'll open a separate issue on that, since I think this may be very useful not just for FIST but in general.

**Comment by juliohm** [comment]
Amazing. Thanks for the organizing these tests. I agree that it would be nice to have some quality metrics for mesh elements 👍🏽 

I will also open other issues on my radar so that we keep things moving in parallel.

**Comment by juliohm** [comment]
Similar to the issue #77 , this issue is research-oriented. We will only find the need to implement such metrics when a bug is encountered in any of our triangulation algorithms. In that case we will fix the bug, and implement a test function as described above to avoid reintroducing the bug in the future. To keep our issue tracker clean with issues that reflect the current needs of the community, I will close this one.
